stathat-json.gemspec

```diff
@@ -28,5 +28,5 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'celluloid-io', '~> 0.17'
   spec.add_dependency 'faraday', '~> 0.9'
   spec.add_dependency 'multi_json', '~> 1.10'
-  spec.add_dependency 'net-http-persistent', '~> 2.9'
+  spec.add_dependency 'net-http-persistent', '>= 3.0.0'
```

Comment:
Since we're here, let's loosen this up as much as possible - not more. Following semver I think we probably should go with `'>= 2.9', '< 4'`

Reply:
Sure thing